nautobot/utilities/git.py

```diff
@@ -8,20 +8,46 @@
 
 logger = logging.getLogger("nautobot.utilities.git")
 
+# 'A' and 'D' status are swapped because of the way the repo.git.diff was implemented
+# e.g. 'A' actually stands for Addition but in this case is Deletion
+GIT_STATUS_MAP = {
+    "A": "Deletion",
+    "M": "Modification",
+    "C": "Copy",
+    "D": "Addition",
+    "R": "Renaming",
+    "T": "File Type Changed",
+    "U": "File Unmerged",
+    "X": "Unknown",
+}
+
+
+def swap_status_initials(x):
+    """Swap github status initials with its equivalent"""
+    return GIT_STATUS_MAP.get(x[0]) + " - " + x[1]
```

Comment:
It's not entirely clear what this is doing. What is `x`? Can we please make this more readable? It appears to be a tuple. Can we please unpack it instead of using indexing?

```suggestion
def swap_status_initials(x):
    """Swap GitHub status initials with its equivalent."""
    return GIT_STATUS_MAP.get(x[0]) + " - " + x[1]
```